Week-5/main.test.js

```diff
@@ -0,0 +1,39 @@
+
+const {MathOperations} = require('./Ex4');
+
+test("sum", () => {
```

Comment:
Explore mocking of functions and promises

Reply:
Noted. Thanks